src/Surfnet/ServiceProviderDashboard/Application/Service/ServiceConnectionService.php

```diff
@@ -119,12 +119,12 @@ private function addEntitiesToCollection(
             $supportContact = $this->composeContactString($entity, 'support');
             $technicalContact = $this->composeContactString($entity, 'technical');
             $adminContact = $this->composeContactString($entity, 'admin');
-            if ($service === null || !in_array($entity->getProtocol()->getProtocol(), $allowedProtocols)) {
-                // Skipping entities of which we do not know the service (team is not set on any of our Services)
+            if (!in_array($entity->getProtocol()->getProtocol(), $allowedProtocols)) {
+                // Skipping entities that are not saml or oidc rp
                 continue;
             }
             if ($entity->getAllowedIdentityProviders()->isAllowAll()) {
-                $serviceName = $service->getOrganizationNameEn();
+                $serviceName = $service?->getOrganizationNameEn() ?: 'Unknown';
```

Comment:
This change is unrelated to this PR. In short this loosens the requirement for the service to be present on the ManageEntity. If the service is not known (the entity is not tracked by SP Dashboard) the entity may still be displayed on the SURFconext representative page. @quartje requested me to categorize these entities under the service 'Unknown' section.

I call upon the boyscout rule here.